lib/modules/search.js

```diff
@@ -166,20 +166,22 @@ function makeOptionSearchResult(moduleKey, optionKey) {
 	const mod = Modules.get(moduleKey);
 	const option = mod.options[optionKey];
 
+	const optionName = i18n(option.title) || _.startCase(option.title || optionKey);
```

Comment:
Could you also add i18n to [rankOption's "get option title"](https://github.com/wojtekmaj/Reddit-Enhancement-Suite/blob/687161bdd5c554f70d7a5f1f0a62baf7f9cc48d3/lib/modules/search.js#L135)